packages/snaps-rpc-methods/src/utils.ts

```diff
@@ -118,6 +113,95 @@ type DeriveEntropyOptions = {
   cryptographicFunctions: CryptographicFunctions | undefined;
 };
 
+type SeedDeriveEntropyOptions = BaseDeriveEntropyOptions & {
+  /**
+   * The mnemonic seed to use for entropy derivation.
+   */
+  seed: Uint8Array;
+};
+
+type MnemonicDeriveEntropyOptions = BaseDeriveEntropyOptions & {
+  /**
+   * The mnemonic phrase to use for entropy derivation.
+   */
+  mnemonicPhrase: Uint8Array;
+};
+
+/**
+ * Get the derivation path to use for entropy derivation.
+ *
+ * This is based on the reference implementation of
+ * [SIP-6](https://metamask.github.io/SIPs/SIPS/sip-6).
+ *
+ * @param options - The options for entropy derivation.
+ * @param options.input - The input value to derive entropy from.
+ * @param options.salt - An optional salt to use when deriving entropy.
+ * @param options.magic - A hardened BIP-32 index, which is used to derive the
+ * root key from the mnemonic phrase.
+ * @returns The derivation path to be used for entropy key derivation.
+ */
+function getEntropyDerivationPath({
+  input,
+  salt,
+  magic,
+}: Required<Omit<BaseDeriveEntropyOptions, 'cryptographicFunctions'>>):
```

Comment:
Why is `cryptographicFunctions` required here if it's not used?

Reply:
It's not required, it is omitted and the remaining elements are made required, such as `salt` which was previously allowed to be undefined with a default value. But I've kept the default value in the calling functions instead

Reply:
Oh, I misread.